spec/requests/api/v1/admin/organizations_spec.rb

```diff
@@ -0,0 +1,120 @@
+require "rails_helper"
+
+RSpec.describe "/api/admin/organizations" do
+  let(:image) { Rails.root.join("app/assets/images/#{rand(1..40)}.png").open }
+  let!(:org_params) do
+    {
+      name: "Test Org",
+      summary: "a test org",
+      url: "https://testorg.io",
+      profile_image: image,
```

Comment:
this is the big code smell currently: we 1) only are able to create an organization when a profile image is present, and 2) can only set a profile image by uploading a file.

The [Organizations controller](https://github.com/forem/forem/blob/main/app/controllers/organizations_controller.rb#L125) takes some careful steps to ensure this too. 

Joshua and I briefly discussed using the backup link as the organization's image in something like a pre-validation hook when no profile image is present.

... but that would require changes in the model to allow for associating an organization with an image via url rather than file upload, it feels like it may be too clunky to require that uplift from the perspective of Forem Tools consumers.